ci-operator/config/windup/windup-ui-tests/windup-windup-ui-tests-main__mtr-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  to: mtr-runner
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: mtr-interop-aws
+  cron: 0 1 * * 1
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: aws.interop.ccitredhat.com
+      MTR_TESTS_UI_SCOPE: interop
+      OPERATORS: |
+        [
+            {"name": "mtr-operator", "source": "redhat-operators", "channel": "alpha", "install_namespace": "mtr", "target_namespaces": "mtr"}
```

Comment:
what did the `source` refers to here?
I was also wondering, in general if there are multiple operators, is possible to add multiple operators lists here? Or we need to do in another ref?

Reply:
Source is the catalog that the operator should be installed from. You can define as many operators here as you'd like. Here is the documentation for that step: https://github.com/openshift/release/blob/master/ci-operator/step-registry/install-operators/README.md